main.go

```diff
@@ -397,6 +434,7 @@ func configureSAML() error {
 	return nil
 }
 
+// BestDomain will attempt to isloate the domain this binary is running from
```

Comment:
@gavinelder Thought you had a PR to comment this out?

Reply:
I did, it will be deleted all together shortly as per https://github.com/subspacecommunity/subspace/pull/42#discussion_r429557138